packages/uniforms-bridge-graphql/src/GraphQLBridge.js

```diff
@@ -126,14 +127,15 @@ export default class GraphQLBridge extends Bridge {
 
     const field = this.getField(nameGeneric, false);
     const fieldType = extractFromNonNull(field).type;
+    const defaultLabel = toHumanLabel(field.name) || '';
```

Comment:
I'm strongly against adding such functionality to the core package. How about using [`_.lowerCase`](https://lodash.com/docs/4.17.15#lowerCase) instead? It's almost it:

```js
function toHumanLabel(label) {
  // label === 'x.y.z.firstNameField';
  label = joinName(null, label);
  // label === ['x', 'y', 'z', 'firstNameField']
  label = label[label.length - 1];
  // label === 'firstNameField'
  label = _.lowerCase(label);
  // 'first name field'
  label = _.upperFirst(label);                     // Option A.
  label = label[0].toUpperCase() + label.slice(1); // Option B.
  // 'First name field'
  return label;
}
```

There's also [`_.startCase`](https://lodash.com/docs/4.17.15#startCase), but I like it a bit less. And let's keep this function local to this file, just like `extractFromNull`. There's no need for explicit tests of it - simply covering it in the other tests (as it is now) is enough.